[Doc] Fix TryGetLabel label typo

gregkalapos opened the pull request:
In the public API doc the label was out-of-sync with the real name of the method.

apmmachine commented:
## :green_heart: Build Succeeded


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-dotnet%2Fapm-agent-dotnet-mbp%2FPR-1088/detail/PR-1088/1//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-dotnet%2Fapm-agent-dotnet-mbp%2FPR-1088/detail/PR-1088/1//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-dotnet%2Fapm-agent-dotnet-mbp%2FPR-1088/detail/PR-1088/1//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-dotnet%2Fapm-agent-dotnet-mbp%2FPR-1088/detail/PR-1088/1//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://apm-agent-dotnet_1088.docs-preview.app.elstc.co/diff)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats

* Build Cause: `Pull request #1088 opened`

* Start Time: 2020-12-14T21:44:27.202+0000

* Duration: 5 min 0 sec





</p>
</details>

<!-- TEST RESULTS IF ANY-->


<!-- STEPS ERRORS IF ANY -->








<!--COMMENT_GENERATED_WITH_ID_comment.id-->

bmorelli25 reviewed (approved):
Oops! At least the code examples were correct. LGTM!

gregkalapos commented:
> Oops! At least the code examples were correct. LGTM!

This it totally on me. This got merged with https://github.com/elastic/apm-agent-dotnet/pull/1057 - we decided to change the method within the PR, I added a ToDo to make sure the doc is in-sync with the final version, I also noticed the ToDo before merge, updated everything ...except the label 🤦 .
